endpoints/auth.go

```diff
@@ -55,56 +54,56 @@ var (
 		return time.Now().UTC()
 	}
 
-	// ContextFactory takes an in-flight HTTP request and creates a new
-	// context.
+	// AuthenticatorFactory creates a new authenticator.
 	//
 	// It is a variable on purpose. You can set it to a stub implementation
 	// in tests.
-	ContextFactory func(*http.Request) Context
+	AuthenticatorFactory func() Authenticator
```

Comment:
It's useful for application integration tests where you want to fake out authorization.